h2/src/main/org/h2/expression/Function.java

```diff
@@ -2049,6 +2056,10 @@ private static int makeRegexpFlags(String stringFlags) {
                     case 'm':
                         flags |= Pattern.MULTILINE;
                         break;
+                    case 'g':
+                        if (ignoreGlobalFlag) {
+                            break;
+                        }
```

Comment:
Please, insert a new line with
```Java
//$FALL-THROUGH$
```
comment here, before the line with `default` operator.

Without it some compilers / IDEs show a warning.